### DISCUSSION TITLE
Microsoft Seceruty reporting Shell as a Trojan

### DISCUSSION THREAD

**Discussion by RafaelLorenzoni** [moudey/Shell#13] [General]
When downloaded or when i Register in the system, the Microsoft Security report this Shell as a trojan.

Trojan:Script/Wacatac.B!ml

**Comment by moudey** [comment]
> When downloaded or when i Register in the system, the Microsoft Security report this Shell as a trojan.
> 
> Trojan:Script/Wacatac.B!ml

Shell does not contain any viruses or malware, and It was checked by virustotal.com and hybrid-analysis.com. mostly this is a false positive detection

virustotal.com result
[shell.exe](https://www.virustotal.com/gui/file/3b2faca90ca4683f0d2bcd6b21dc20380a7cf8bac2124f88ce91dcc842dbec9e)
[shell.dll](https://www.virustotal.com/gui/file/e918d29fc2fddbc9bf5ba32638eec61cec480931f50ff6986892c397b9eb6f03)

hybrid-analysis.com result
[shell.exe](https://hybrid-analysis.com/sample/3b2faca90ca4683f0d2bcd6b21dc20380a7cf8bac2124f88ce91dcc842dbec9e)
[shell.dll](https://hybrid-analysis.com/sample/e918d29fc2fddbc9bf5ba32638eec61cec480931f50ff6986892c397b9eb6f03)

**Comment by moudey** [comment]
Please report Shell as safe if possible

**Comment by win98se** [comment]
@RafaelLorenzoni @moudey Both of you can try to submit the program for clarification as a home customer and a software developer respectively at https://www.microsoft.com/en-us/wdsi/filesubmission.

Macrohard shall not mark it as a trojan anymore, as it is a false positive and does not harm Windows in any way.

**Comment by moudey** [comment]
Result in Microsoft review No malware was detected "setup.exe" :)

![Web capture_29-8-2022_194741_www microsoft com](https://user-images.githubusercontent.com/39198/187265477-252ab19d-9667-41d2-bbea-b23e5a66e661.jpeg)

**Comment by moudey** [comment]
Result in Microsoft review No malware was detected "shell.exe, shell.dll" :)

![Web capture_29-8-2022_222114_www microsoft com](https://user-images.githubusercontent.com/39198/187291602-66d97506-3139-46d9-900f-2d4880d17783.jpeg)
